ifstate: configure sysctl on all hosts

MarcelCoding opened the pull request.

MarcelCoding committed:
ifstate: configure sysctl on all hosts

tanneberger reviewed (approved):
looks sensible. 

This is for the as112 and audix VM which are connected to the peering lan?

MarcelCoding commented:
This is for all vm's

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'nixpkgs':
    'github:NuschtOS/nuschtpkgs/a61900ad3724a9854282dbebae54902f8ad44136?narHash=sha256-aelxLnG4G8nMwpM3J8uK5qzieZ78WgYXQE%2B%2BADNso18%3D' (2025-04-04)
  → 'github:NuschtOS/nuschtpkgs/0ba4ea69e30366047db2307d324cbab50efffd09?narHash=sha256-80Y0ElVnd42MdVAM2jyKe5Cg4Jnh8YOxm%2B5ZyiXrsrc%3D' (2025-04-06)
• Updated input 'sops-nix':
    'github:mic92/sops-nix/cff8437c5fe8c68fc3a840a21bf1f4dc801da40d?narHash=sha256-2b11EYa08oqDmF3zEBLkG1AoNn9rB1k39ew/T/mSvbU%3D' (2025-04-04)
  → 'github:mic92/sops-nix/523f58a4faff6c67f5f685bed33a7721e984c304?narHash=sha256-zr2jmWeWyhCD8WmO2aWov2g0WPPuZfcJDKzMJZYGq3Y%3D' (2025-04-06)

MarcelCoding committed:
nextcloud: fix systemd dependencies

MarcelCoding committed:
onlyoffice: 8.3.0.1 -> 8.3.2.1
